runtime/bcutil/ClassFileOracle.cpp

```diff
@@ -1734,6 +1734,13 @@ ClassFileOracle::walkMethodCodeAttributeCode(U_16 methodIndex)
 			}
 			break;
 		}
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+		case CFR_BC_defaultvalue:
+			cpIndex = PARAM_U16();
+			addBytecodeFixupEntry(entry++, codeIndex + 1, cpIndex, ConstantPoolMap::NEW);
```

Comment:
Instead of using `ConstantPoolMap::NEW`, should I define a `ConstantPoolMap::DEFAULT_VALUE = SPLIT1`?

Reply:
yes